src/components/Votings.tsx

```diff
@@ -0,0 +1,72 @@
+import { Scopes, useState } from '@state-less/react-server';
+import { ServerSideProps } from './ServerSideProps';
+
+type VotingObject = {
+    title: string;
+    upvotes: number;
+    downvotes: number;
+    key?: string;
+};
+
+type ScoreObject = {
+    leftBound: number;
+    rightBound: number;
+};
+
+export const Votings = () => {
+    const [voting, setVoting] = useState<VotingObject>(
+        {
+            title: 'Voting',
+            upvotes: 0,
+            downvotes: 0,
+        },
+        {
+            key: 'votings',
+            scope: Scopes.Global,
+        }
+    );
+    const [score, setScore] = useState<ScoreObject>(
+        {
+            leftBound: 0,
+            rightBound: 0,
+        },
+        {
+            key: 'score',
+            scope: Scopes.Global,
+        }
+    );
+
+    const upvote = () => {
+        setVoting({ ...voting, upvotes: voting.upvotes + 1 });
+    };
+
+    const downvote = () => {
+        setVoting({ ...voting, downvotes: voting.downvotes + 1 });
+    };
+
+    const wilsonScoreInterval = () => {
```

Comment:
Awesome! Just a minor thought. You can simply call `wilsonScoreInterval` in the voting functions e.g. 
```
 const downvote = () => {
    const newVoting = {...voting, downvotes: voting.downvotes + 1};
    setVoting(newVoting);
    wilsonScoreInterval(newVoting) //just pass the new state to the function (if it's not available in the scope)
};
```

This way you don't have to call it from the client after each update and you save a whole roundtrip.

Reply:
Yeah I understand. I have made all the required changes.